changelog.md

```diff
@@ -1,5 +1,80 @@
+- Fix: Autoreplace rail/road list only listed buildable types (#13887)
+- Fix: [NewGRF] Display an error, if NewGRF reference out-of-bounds string parameters in gender/plural choices (#13881)
+- Fix #13849: Settings in old saves could be overridden by defaults (#13874)
+- Fix #13562: Removed cost estimation message from money cheat (#13857)
+- Fix: [NewGRF] Plurals and genders did not work in strings with cases or substrings (#13853, #13852)
+- Fix: [NewGRF] String parameter stack and case selection were not processed for control code 0x81 (#13851)
+- Fix #13839: Incorrect colour of first company legend in smallmap window (#13841)
+- Fix: i circumflex width in TrueType small font (#13836)
+- Fix: Don't show owner of non-existent road (#13824)
+- Fix: Error message window timeout doesn't match setting (#13812)
+- Fix #13795: Crash in vehicle list of 32-bit platforms (#13796)
+- Fix: [Script] Company rename event sometimes had the wrong name (#13794)
+- Fix: Improve manager face randomisation (#13776)
+- Fix #13740: [Script] Handle implicit orders for jump orders (#13753)
+- Fix #13749: Default service intervals were not updated when changing timekeeping unit (#13751)
+- Fix #13725: Use proper query strings for changing timetable values (#13737)
+- Fix #11226: Don't draw story page elements that won't be visible (#13736)
+- Fix: More AI than max_no_competitors could start with competitors_interval=0 (#13670)
+- Fix: League table window ignored the minimal size in its widget description (#13629)
+- Fix: Incorrect snow density when making rocks snowy (#13626)
+- Fix: NewGRF vehicles display loading sprites when not actually loading or unloading (#13554)
+- Fix #12925: Prevent cost estimates for settings changes (#13550)
+- Fix: [Script] Report errors happening during 'Load()' (#13537)
+- Fix: [Script] Improve type checking of parameters (#13522)
+- Fix: [Script] Don't set CommandCallback for asynchronous commands (#13501)
+- Fix: Missing error messages with sell- and autoreplace-all commands (#13469)
+- Fix: Too many trees when generating trees at same height (#13460)
+- Fix #12912: Company inaugurated year in wallclock mode was not saved (#13448)
+- Fix: [Script] Wrong return value for failed preconditions Vehicle::CloneVehicle (#13445)
+- Fix #13140: Scale initial industry production estimate by cargo scale (#13427)
+- Fix #13384: Crash when remove bus/truck stop tool used on road waypoints (#13391)
+- Fix #12987: Historical houses now always spawn completed (#13332)
+- Fix: [Win32] Font detection didn't work for locales not supporting code pages (#13306)
+- Fix: Restore ability to disable service interval (#13281)
+- Fix: Hide company settings from console commands (#13269)
+- Fix: Disable service interval widgets for non-owned vehicles (#13260)
+- Fix #13225: Cargo payment graph key toggled wrong data sets (#13226)
+- Fix: Rail station tile flags were not set early enough (#13203)
```

Comment:
Not really player-facing, unless I'm reading the PR wrong.

```suggestion
```

Reply:
It's fixing a crash triggered by an user loading an old save game.

Reply:
It fixes a potential crash when loading saves, but that would only have occurred for a short time in the master branch, not since 14.x

Reply:
It's after 15.0-beta1 though, and I'm not seeing anything that would've introduced this issue after 15.0-beta1. So it's a fix of an issue in release, yes it's a beta but so are many of the other bug fixes in here.

Reply:
Fair, Nov 15th to Dec 27th is actually longer than I remembered, and so yes that affects 15.0-beta1.